charts/logan/values.yaml

```diff
@@ -424,6 +434,125 @@ fluentd:
             # Logging Analytics log source to use for parsing and processing Linux YUM Logs.
             ociLALogSourceName: "Linux YUM Logs"
 
+   # Configuration for AWS EKS Control Plane logs like API Server, Audit, Authenticator etc.
+   eksControlPlane:
+      # Collection Type (cloudwatch or s3)
+      collectionType: "cloudwatch"
+      # AWS region
+      region:
+      # Use AssumeRoleCredentials (https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/AssumeRoleCredentials.html) to authenticate
+      # Default is true. Set to false to use access keys
+      awsUseSts: true
+      # The role ARN to assume when using AWS Security Token Service authentication
+      awsStsRoleArn:
+      # AWS access key and secret access key, needed only when awsUseSts is explicitly set to false
+      #awsKeyId:
+      #awsSecKey:
+      # CloudWatch Log Group name of the EKS cluster. Automatically determined by extracting cluster name from kubernetesClusterId and
+      # following naming syntax as "aws/eks/<cluster name>/cluster". Below field can be used to override this behavior.
+      #cwLogGroupName:
+      # S3 related settings
+      # S3 bucket name to which EKS Control Plane logs are being streamed using a subscription filter
+      s3Bucket:
+      ociLALogGroupID:
+      #metadata:
+         #"Client Host Region": "America"
+         #"Environment": "Production"
+         #"Third Key": "Third Value"
+      #ociLAEntityID:
+      # Worker number in case of multi process workers enabled. If not set when multi process workers enabled, then it defaults to 0.
+      #worker:
+      logs:
+         # If using cloudwatch collection mechanism, apiserver and audit logs need to be part of the same worker as they share the same log stream name prefix.
+         # Thus "worker" variable is only picked up from "apiserver" section.
+         apiserver:
+            # CloudWatch Log Stream name
+            cwLogStreamName: "kube-apiserver"
+            # SQS queue name which is notified when apiserver log object is created in S3 bucket
+            sqsQueue: "apiserver"
+            # S3 object key
+            objectKey: .*?kube-apiserver/
+            # Logging Analytics log source to use for parsing and processing EKS Control Plane API Server Logs.
+            ociLALogSourceName: "Kubernetes API Server Logs"
+            multilineStartRegExp: /^(\S)(\d{2})(\d{2})/
```

Comment:
In other places we are using, /^\S\d{2}\d{2}\s+[^\:]+:[^\:]+:[^\.]+\.\d{0,3}/. Will the current start expression sufficient in general ?

Reply:
Updated to be more specific